roles/k8s_best_practices_certsuite/templates/certsuite_config.yml.j2

```diff
@@ -50,6 +50,15 @@ servicesignorelist:
   - {{ item }}
 {% endfor %}
 
+{% if kbpc_cwe_apikey is defined and kbpc_cwe_project_id is defined %}
+connectAPIConfig:
+  baseURL: "https://access.redhat.com/hydra/cwe/rest/v1.0"
+  apiKey: {{ kbpc_cwe_apikey }}
+  projectID: {{ kbpc_cwe_project_id }}
```

Comment:
If this parameter is required, it should be defined in the README how to obtain it and how to provide it

Reply:
Just an FYI, essentially the way the logic works in the certsuite is it will only attempt the tarball upload if the two values for `apiKey` and `projectID` are set.  I guess you could say they are required but nothing will fail if they are not supplied you just won't see an upload happen.

Reply:
the parameter is optional, but I'll update the README